src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip

```diff
@@ -3,45 +3,34 @@
 
 @snippet generate(sampleFile)
    //// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "{@sampleFile.className}" ]
-   //// STUB standalone sample "{@sampleFile.className}" /////
-
-   // FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.
-
+  'use strict';
```

Comment:
Use the metadata line like we do in the other languages:
```
@// DO NOT EDIT! This is a generated sample ("{@sample.callingForm}",  "{@sample.valueSet.id}")
```

We should have this be consistent to facilitate any tooling that may need to be built on top. If we decide to change the format, we should do uniformly for all langs.